test/Groups/homomorphisms.jl

```diff
@@ -244,6 +244,21 @@ end
       end
    end
 
+   @testset "Infinite GrpAbFinGen to GAPGroup" begin
+      Agens = matrix(ZZ, 2, 2, [2, 3, 0, 0])
+      A = abelian_group(Agens)
+      for T in [FPGroup]
+         iso = @inferred isomorphism(T, A)
+         for x in gens(A)
+            for y in gens(A)
```

Comment:
Maybe more julia-ish would be this?
```suggestion
         for x in gens(A), y in gens(A)
```

Reply:
Good idea (also in other instances of nested loops in test files)